src/extensions/extension-loader.ts

```diff
@@ -51,6 +51,18 @@ export class ExtensionLoader {
     return extensions;
   }
 
+  // Transform userExtensions to a state object for storing into ExtensionsStore
+  @computed get storeState() {
+    return Array.from(this.userExtensions).reduce((state, [extId, ext]) => {
+      state[extId] = {
+        enabled: ext.isEnabled,
+        name: ext.manifest.name,
+      };
+
+      return state;
+    }, {} as Record<LensExtensionId, LensExtensionState>);
+  }
+
   @action
   async init() {
```

Comment:
I guess in the future, this really shouldn't be a class and instead some functions since we really only should "init" once.